Plugins/Skinning/src/PositionBasedSkinning/ParticleData.h

```diff
@@ -0,0 +1,247 @@
+#ifndef __PARTICLEDATA_H__
+#define __PARTICLEDATA_H__
+
+#include "Config.h"
+#include <Core/Math/LinearAlgebra.hpp>
+#include <vector>
+
+
+namespace PBD
```

Comment:
Did you rewrite the whole pbd lib? or is it a simplified/enhanced version?
It would be better to have the pbd lib by itself as a submodule (either from some pbd repo or from a self-maintained repo) and link to it. Thus others may be able to use it as they please.
This might be the most important change to work on!